addons/l10n_lv/data/account_account_tag_data.xml

```diff
@@ -0,0 +1,1504 @@
+    <field name="applicability">accounts</field>
+    <field name="color" eval="8"/>
+  </record>
+
+  <record id="account_account_tag_pza2_190" model="account.account.tag">
+    <field name="name">PZA2.190. b) from other persons</field>
+    <field name="applicability">accounts</field>
+    <field name="color" eval="8"/>
+  </record>
+
+  <record id="account_account_tag_pza2_200" model="account.account.tag">
+    <field name="name">PZA2.200. Impairment allowances on long-term and short-term
+      financial investments</field>
+    <field name="applicability">accounts</field>
+    <field name="color" eval="8"/>
+  </record>
+
+  <record id="account_account_tag_pza2_210" model="account.account.tag">
+    <field name="name">PZA2.210. Interest and similar costs</field>
+    <field name="applicability">accounts</field>
+    <field name="color" eval="8"/>
+  </record>
+
+  <record id="account_account_tag_pza2_220" model="account.account.tag">
+    <field name="name">PZA2.220. a) radniecīgajām sabiedrībām</field>
+    <field name="applicability">accounts</field>
+    <field name="color" eval="8"/>
+  </record>
+
+  <record id="account_account_tag_pza2_230" model="account.account.tag">
+    <field name="name">PZA2.230. b) from other persons</field>
+    <field name="applicability">accounts</field>
+    <field name="color" eval="8"/>
+  </record>
+
+  <record id="account_account_tag_pza2_240" model="account.account.tag">
+    <field name="name">PZA2.240. Profit or loss before corporate tax</field>
+    <field name="applicability">accounts</field>
+    <field name="color" eval="8"/>
+  </record>
+
+  <record id="account_account_tag_pza2_250" model="account.account.tag">
+    <field name="name">PZA2.250. Corporate income tax for the current year</field>
+    <field name="applicability">accounts</field>
+    <field name="color" eval="8"/>
+  </record>
+
+  <record id="account_account_tag_pza2_260" model="account.account.tag">
+    <field name="name">PZA2.260. Profit or loss after corporate tax</field>
+    <field name="applicability">accounts</field>
+    <field name="color" eval="8"/>
+  </record>
+
+  <record id="account_account_tag_pza2_270" model="account.account.tag">
+    <field name="name">PZA2.270. Income or expenses from changes in deferred tax asset or
+      liability balances</field>
+    <field name="applicability">accounts</field>
+    <field name="color" eval="8"/>
+  </record>
+
+  <record id="account_account_tag_pza2_280" model="account.account.tag">
+    <field name="name">PZA2.280. Extraordinary dividends</field>
+    <field name="applicability">accounts</field>
+    <field name="color" eval="8"/>
+  </record>
+
+  <record id="account_account_tag_pza2_290" model="account.account.tag">
+    <field name="name">PZA2.290. Profit or loss for the current year</field>
+    <field name="applicability">accounts</field>
+    <field name="color" eval="8"/>
+  </record>
+
+  <!-- Tags for taxes -->
+
+  <!-- VAT report 1.1. types -->
```

Comment:
The account tags for the VAT report should be defined directly in the report definition, in the `account.report.line`. 
See https://github.com/odoo/odoo/blob/b7232b14b7f2423241b24b4eef1300f1a1b425cf/addons/l10n_bg/data/tax_report.xml#L31-L34 for an example - this creates a tax tag with xmlid "l10n_bg_tax_report_11_tag" and which corresponds to the tax grid "11".